docs/README.md

```diff
@@ -0,0 +1,63 @@
+# Build the `cleanlab` docs **locally**
```

Comment:
Before this, can you also specify how the docs are being built via Github actions during PRs as well.

Reply:
otherwise love this readme!

Reply:
Got it! have added this right at the top of the README. screenshot attached, mock-up [here](https://github.com/weijinglok/cleanlab/tree/patch_v1.1/docs)
![image](https://user-images.githubusercontent.com/90272361/155888335-89a10fa8-8190-4f0b-945d-5ebea7ca65ae.png)